support/proxy/proxy.go

```diff
@@ -0,0 +1,63 @@
+package proxy
+
+import (
+	"os"
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+func SetEnvVars(env *[]corev1.EnvVar, additionalNoProxy ...string) {
+	SetEnvVarsTo(env,
+		os.Getenv("HTTP_PROXY"),
+		os.Getenv("HTTPS_PROXY"),
+		os.Getenv("NO_PROXY")+","+strings.Join(append(additionalNoProxy, "kube-apiserver"), ","),
```

Comment:
should we include localhost here?

Reply:
I guess we can but we are not doing loopback connections right now